src/servers/database.ts

```diff
@@ -550,7 +588,49 @@ export class DatabaseServer extends SkeletonServer {
       return null;
     }
   }
+  async eth_getTransactionReceiptsByBlockNumber(
+    _request: any,
+    blockNumber: web3.Quantity | web3.Tag
+  ): Promise<web3.TransactionReceipt[] | null> {
+    const blockNumber_ =
+      parseBlockSpec(blockNumber) || (await this._fetchCurrentBlockID());
+    try {
+      const {
+        rows: [receipt],
+      } = await this._query(
+        `SELECT
```

Comment:
Would be good to move this query to the `etc/schema/functions` as a separate sql function.